src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsauditapi/jpa/AuditRepositoryTest.kt

```diff
@@ -243,11 +287,67 @@ class AuditRepositoryTest {
         null,
         null,
         null,
-        who = null,
+        null,
       )
       assertThat(auditEvents.size).isEqualTo(3)
     }
 
+    @Test()
+    internal fun `filter audit events by an invalid start date, throws exception`() {
+      auditRepository.save(
+        AuditEvent(
+          what = "An Event",
+          `when` = Instant.now(),
+          operationId = "123456789",
+          service = "Service-A",
+          who = "John Smith",
+        )
+      )
+
+      assertThat(auditRepository.count()).isEqualTo(1)
+
+      assertThrows(
+        DateTimeParseException::class.java
+      ) {
+        auditRepository.findPage(
+          Pageable.unpaged(),
+          Instant.parse("2021-14-04T17:17:30Z"),
+          null,
+          null,
+          null,
+          null,
+        )
+      }
+    }
+
+    @Test()
+    internal fun `filter audit events by an invalid end date, throws exception`() {
+      auditRepository.save(
+        AuditEvent(
+          what = "An Event",
+          `when` = Instant.now(),
+          operationId = "123456789",
+          service = "Service-A",
+          who = "John Smith",
+        )
+      )
+
+      assertThat(auditRepository.count()).isEqualTo(1)
+
+      assertThrows(
+        DateTimeParseException::class.java
+      ) {
+        auditRepository.findPage(
+          Pageable.unpaged(),
+          null,
+          Instant.parse("2021-14-74T17:17:30Z"),
```

Comment:
This test is just testing that Instant.parse works - it needs to be part of the e2e tests where an invalid date/instant is passed in